boards/arm/frdm_k82f/frdm_k82f_defconfig

```diff
@@ -11,6 +11,7 @@ CONFIG_PINMUX=y
 CONFIG_GPIO=y
 CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=120000000
 CONFIG_OSC_LOW_POWER=y
+CONFIG_COUNTER=y
```

Comment:
This will enable the counter in every application for this board, including blinky and hello_world. Please remove.

Reply:
ok